authentication/views.py

```diff
@@ -73,4 +73,15 @@ def get(
         """
         GET endpoint for logging a user in.
         """
-        return redirect(get_redirect_url(request))
+        redirect_url = get_redirect_url(request)
+        if not request.user.is_anonymous:
+            profile = request.user.profile
+            if (
+                not profile.completed_onboarding
+                and request.GET.get("skip_onboarding", "0") != "1"
+            ):
+                params = urlencode({"next": redirect_url})
+                redirect_url = f"{settings.MITOL_NEW_USER_LOGIN_URL}?{params}"
+            profile.completed_onboarding = True
+            profile.save()
```

Comment:
I think we only want to mark them as having completed onboarding if the user was redirected there. Otherwise if a request specifies `skip_onboarding` (presumably because we have a redirect url we want them to go straight to), they'll at least be prompted for the onboarding once when they return to Learn directly.

Reply:
Updated the logic such that the flag will only get set at point of redirect to the onboarding page